cmd/metrics-v2.go

```diff
@@ -225,10 +225,20 @@ const (
 	linkOfflineDuration       MetricName = "link_offline_duration_seconds"
 	linkDowntimeTotalDuration MetricName = "link_downtime_duration_seconds"
 
-	avgInQueueCount MetricName = "average_queued_count"
-	avgInQueueBytes MetricName = "average_queued_bytes"
-	maxInQueueCount MetricName = "max_queued_count"
-	maxInQueueBytes MetricName = "max_queued_bytes"
+	avgInQueueCount        MetricName = "average_queued_count"
+	avgInQueueBytes        MetricName = "average_queued_bytes"
+	maxInQueueCount        MetricName = "max_queued_count"
+	maxInQueueBytes        MetricName = "max_queued_bytes"
+	proxiedGetReqTotal     MetricName = "proxied_get_requests_total"
+	proxiedHeadReqTotal    MetricName = "proxied_head_requests_total"
+	proxiedPutTagReqTotal  MetricName = "proxied_put_tagging_requests_total"
+	proxiedGetTagReqTotal  MetricName = "proxied_get_tagging_requests_total"
+	proxiedRmvTagReqTotal  MetricName = "proxied_delete_tagging_requests_total"
+	proxiedGetReqFailed    MetricName = "proxied_get_requests_failures"
+	proxiedHeadReqFailed   MetricName = "proxied_head_requests_failures"
+	proxiedPutTagReqFailed MetricName = "proxied_put_tagging_requests_failures"
+	proxiedGetTagReqFailed MetricName = "proxied_get_tagging_requests_failures"
```

Comment:
```suggestion
	proxiedGetReqFailed    MetricName = "proxied_get_requests_failed"
	proxiedHeadReqFailed   MetricName = "proxied_head_requests_failed"
	proxiedPutTagReqFailed MetricName = "proxied_put_tagging_requests_failed"
	proxiedGetTagReqFailed MetricName = "proxied_get_tagging_requests_failed"
```

Match the name of the metric